src/js/view/controls/controlbar.js

```diff
@@ -49,45 +47,24 @@ function createCastButton(castToggle, localization) {
         SimpleTooltip(airplayButton.element(), 'airplay', localization.airplay);
 
         return airplayButton;
+    } else if (Browser.chrome || OS.iOS) {
```

Comment:
```
    if (!Browser.chrome || OS.iOS) {
        return;
    }
```
was the previous code, should this be `&& !OS.iOS`

Reply:
Thanks! Changed it to use the same check we use to webpack load the Chromecast controller in jwplayer-commercial.